packages/client/components/promptResponse/PromptResponseEditor.tsx

```diff
@@ -65,6 +65,14 @@ const PromptResponseEditor = (props: Props) => {
     editable: !readOnly
   })
 
+  useEffect(() => {
+    if (!editor) {
+      return
+    }
+
+    editor.commands.setContent(content)
+  }, [editor, content])
```

Comment:
this is needed as the `content` specified [here](https://github.com/ParabolInc/parabol/pull/6504/files#diff-b3416557ebc46ac9b7d1fa9ef62f5564cf40767ea9895e098715c22cb32d4549R60) is just the initial content. without that, when the relay store is updated during the subscription event or the mutation, the editor won't update.

Reply:
+1 Thoughts on using the dependency list from `useEditor` (see https://github.com/ParabolInc/parabol/pull/6469/commits/1bdb26fd6af9b2eb525473490c5ce3313cd5de24#diff-b3416557ebc46ac9b7d1fa9ef62f5564cf40767ea9895e098715c22cb32d4549R68)?  I'm not sure what the tradeoffs are, but that seems cleaner in the context of hooks.

Reply:
how did I miss that? thanks for pointing that out! `useEffect` should really be the last resort, this is much better.